### ISSUE TITLE
Have you used the add-on with a 2nd Joplin profile?

### ISSUE THREAD

**Issue by Dricc123** [marph91/thunderbird-joplin-export#27]
To switch Joplin profiles, I would need to go through the 'Enable the Webclipper' process again, correct? And to do that I would need to have the webclipper for my 2nd Joplin profile active, correct? active in another browser, I suppose. I'm curious to know how complex the procedure is when playing with two profiles.

**Comment by Dricc123** [comment]
because another option would be to use the add-on with my 1st profile, then export the notes to MD+FrontMatter, and import into my 2nd profile.

**Comment by marph91** [comment]
I only use a single profile. So you need to try or ask in [the forum](https://discourse.joplinapp.org/). It sounds a bit complicated, since you could create the frontmatter directly in the Thunderbird add-on.

**Comment by Dricc123** [comment]
> It sounds a bit complicated, since you could create the frontmatter directly in the Thunderbird add-on.

could you rephrase this, as I'm not sure I follow you.

**Comment by marph91** [comment]
> >  It sounds a bit complicated, since you could create the frontmatter directly in the Thunderbird add-on.
> 
> could you rephrase this, as I'm not sure I follow you.

Maybe I misunderstood. I thought you want to export MD+FrontMatter just to create the frontmatter. If that's not the case, you can ignore my remark.

**Comment by Dricc123** [comment]
Thanks. Feel free to close this issue. I don't need the use with a 2nd Joplin profile at the end.
